app/app.go

```diff
@@ -218,6 +223,7 @@ var (
 		govtypes.ModuleName:            {authtypes.Burner},
 		ibctransfertypes.ModuleName:    {authtypes.Minter, authtypes.Burner},
 		wasm.ModuleName:                {authtypes.Burner},
+		rewardtypes.ModuleName:         nil,
```

Comment:
check this is mandatory

Reply:
Used as the first module account initial